packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/templates/expandableSearchBoxInput.html

```diff
@@ -46,9 +46,10 @@
                 >
                   <lightning-button-icon
                     onclick={clearInput}
+                    onkeydown={handleKeyDownOnClearButton}
                     icon-name="utility:close"
-                    class="slds-grid searchbox__clear-button"
                     variant="bare"
+                    class="slds-grid searchbox__clear-button"
```

Comment:
What is the effect of adding this class?

Reply:
it was added in a separate PR mostly for the unit tests as a selector, but the change here is just the order in which it appears, I changed it so that the attributes are in the same order in both default and expandable searchBoxInput templates.